.teamcity/settings.kts

```diff
@@ -105,6 +106,18 @@ fun Project.buildAll(versionBuild: BuildType) = BuildType {
         }
     }
 
+    features {
```

Comment:
Teamcity build configuration files are auto-generated by infra plugin. Please add any additional configurations to the `additionalConfiguration.kt` file.

You should configure `knownBuilds.buildAll` there.